Testing/ParticlesTests/ParticlesTests.cpp

```diff
@@ -27,6 +28,23 @@ const std::vector<std::string> filenames = {
   test_dir + "seg.ellipsoid_21.isores.pad.com.aligned.cropped.tpSmoothDT_world.particles"
 };
 
+TEST(ParticlesTests, vnlVector)
+{
+  ParticleSystem particleSystem(filenames);
+  ParticleShapeStatistics stats;
+  stats.DoPCA(particleSystem);
+  stats.PrincipalComponentProjections();
+  Eigen::MatrixXd pcaVec = stats.PCALoadings();
+  for(int i=0; i<pcaVec.rows(); i++)
+  {
+    std::cout << "\n";
+    for(int j=0; j<pcaVec.cols(); j++)
+    {
+      std::cout << pcaVec(i,j) << " ";
```

Comment:
Tests should not print anything to the console.

Reply:
Removed